examples/nlp/lasertagger/README.md

```diff
@@ -0,0 +1,130 @@
+# LaserTagger Summarization Example
+
+LaserTagger is a text-editing model which predicts a sequence of token-level
+edit operations to transform a source text into a target text. The model
+currently supports four different edit operations:
+
+1. *Keep* the token.
+2. *Delete* the token.
+3. *Add* a phrase before the token.
+4. *Swap* the order of input sentences (if there are two of them).
+
+Operation 3 can be combined with 1 and 2. Compared to sequence-to-sequence
+models, LaserTagger is (1) less prone to hallucination, (2) more data efficient,
+and (3) faster at inference time.
+
+A detailed method description and evaluation can be found in our EMNLP'19 paper:
+[https://arxiv.org/abs/1909.01187](https://arxiv.org/abs/1909.01187)
```

Comment:
Hmmm, exactly whom do you mean by "our" here ...?

Reply:
Oh, should change it!

Reply:
;)

But this raises another issue: does this PR contain any 3rd party code?

Reply:
yes, the part in the folder official_lasertagger contains code from https://github.com/google-research/lasertagger, which has been modified from TensorFlow compatibility to NeMo PyT compatible code.

Reply:
1) just to make sure: do you have SWIPAT for this?

2) the files containing copied/adapted code should have original headers, the original apache v2 license requires that, moreover I suggest you indicate what are the sources. Two examples how to do it:

Less strict:
https://github.com/NVIDIA/NeMo/blob/e095496e1f24c60eeef0c6920af5a7fceb1f06e0/nemo/collections/nlp/nm/non_trainables/dialogue_state_tracking/template_nlg_multiwoz.py#L18

More strict:
https://github.com/NVIDIA/NeMo/blob/e095496e1f24c60eeef0c6920af5a7fceb1f06e0/nemo/collections/cv/modules/trainables/image_encoder.py#L37

Reply:
1. yep, SWIPAT was approved. 

2. All the copied/adapted code have their original headers, I'll add the strict indication now to all of these sources.

Reply:
Ok, please remember to add "nvidia apache" header aside of "original" one...

Reply:
yep, sure.